src/transformers/models/whisper/modeling_flax_whisper.py

```diff
@@ -1354,46 +1355,63 @@ def generate(
             generation_config = self.generation_config
 
         if return_timestamps is not None:
-            generation_config.return_timestamps = return_timestamps
-
-        if task is not None:
-            generation_config.task = task
+            if not hasattr(generation_config, "no_timestamps_token_id"):
+                raise ValueError(
+                    "You are trying to return timestamps, but the generation config is not properly set."
+                    "Make sure to initialize the generation config with the correct attributes that are needed such as `no_timestamps_token_id`."
+                    "For more details on how to generate the approtiate config, refer to https://github.com/huggingface/transformers/issues/21878#issuecomment-1451902363"
+                )
 
-        if is_multilingual is not None:
```

Comment:
it seems like `is_multilingual` is no longer used -- shall we add a warning, in case the user sets it?

Reply:
It is still used un the default behavior of the flax Timestamp logits processor, let's leave it as is!